src/engine/source/builder/builders/stageBuilderNormalize.cpp

```diff
@@ -38,7 +38,7 @@ Builder getStageNormalizeBuilder(std::shared_ptr<Registry> registry)
         if (!jsonDefinition.isArray())
         {
             throw std::runtime_error(fmt::format(
-                "Invalid json definition type: expected \"array\" but got \"{}\"",
+                R"(Invalid json definition type: expected "array" but got "{}")",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings